app/ui-sidenav/client/sidebarHeader.js

```diff
@@ -316,18 +316,24 @@ Template.sidebarHeader.events({
 	'click .sidebar__header .avatar'(e) {
 		if (!(Meteor.userId() == null && settings.get('Accounts_AllowAnonymousRead'))) {
 			const user = Meteor.user();
-
+			const STATUS_MAP = [
+				'offline',
+				'online',
+				'away',
+				'busy',
+			];
 			const userStatusList = Object.keys(userStatus.list).map((key) => {
 				const status = userStatus.list[key];
-				const customName = status.localizeName ? null : status.name;
 				const name = status.localizeName ? t(status.name) : status.name;
 				const modifier = status.statusType || user.status;
+				const defaultStatus = STATUS_MAP.includes(status.id);
+				const statusText = defaultStatus ? '' : user.statusText || name || t(user.status);
```

Comment:
`name` should come before `user.statusText` here, or it breaks the admin-defined extra status.

Reply:
Oh, but then it causes regular statuses to overwrite the custom ones. We'll need to change the logic here a bit.

Reply:
```const statusText = defaultStatus ? null : status.name;```